corehq/apps/app_manager/views/forms.py

```diff
@@ -283,6 +283,8 @@ def should_edit(attribute):
                     pass
             if xform:
                 save_xform(app, form, xform)
+                case_references = json.loads(request.POST.get('references', "{}"))
+                _update_case_refs_from_form_builder(form, case_references)
```

Comment:
ignore if this comment doesn't make sense since I don't know this code well, but if there's a set of stuff that always needs to happen when updating a form should we pull it out into a shared function so we don't have to remember to update two places?